bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/SPTargetPio.java

```diff
@@ -89,9 +88,8 @@ public static ParamSet getParamSet(final SPTarget spt, final PioFactory factory)
             final HmsDegTarget t = (HmsDegTarget) target;
             Pio.addParam(factory, paramSet, _SYSTEM, t.getTag().tccName);
             paramSet.addParam(t.getEpoch().getParam(factory, _EPOCH));
-            Pio.addParam(factory, paramSet, _BRIGHTNESS, t.getBrightness());
-            Pio.addParam(factory, paramSet, _C1, t.c1ToString());
-            Pio.addParam(factory, paramSet, _C2, t.c2ToString());
+            Pio.addParam(factory, paramSet, _C1, t.getRa().toString());
+            Pio.addParam(factory, paramSet, _C2, t.getDec().toString());
```

Comment:
I suppose we would migrate the name of the PIO param as well here but since this is going away anyway there's no real need?